bosh-create-env-deployment.yml

```diff
@@ -53,6 +53,7 @@ networks:
   - range: (( grab terraform_outputs.private_subnet_az1_cidr ))
     gateway: (( grab terraform_outputs.private_subnet_az1_gateway ))
     dns:
+    - (( grab meta.bosh_static_ip ))
```

Comment:
It looks like the terraform outputs used here come from the tooling stack, meaning this config would point master bosh dns at tooling bosh. Is that right/expected?

Reply:
```
meta:
  bosh_static_ip: (( grab terraform_outputs.master_bosh_static_ip ))
```
The correct IP should be referenced, since this is grabbing the version in meta.